### PR TITLE
added routing example

### PR DESCRIPTION
**Pull request by awebberCentrica**

### CODE REVIEW

File: tests/unit/user/route-test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  setupTest(hooks);
+
+  test('it exists', function (assert) {
+    const route = this.owner.lookup('route:user');
+    assert.ok(route);
+  });
+
+  test('navigate function', function (assert) {
+    const route = this.owner.lookup('route:user');
+    const stub = sinon.stub(route, 'transitionTo');
```
Comment (Leooo) [inline tests/unit/user/route-test.js]: use `sinon.spy` instead I think: `sinon.stub` will totally replace the `transitionTo` method with a sinon Object, I think - hence the error

Comment (Leooo) [reply tests/unit/user/route-test.js]: examples in https://github.com/ConnectedHomes/ember-commons/search?q=sinon.spy&unscoped_q=sinon.spy

Comment (Leooo) [reply tests/unit/user/route-test.js]: (or you have `sinon.stub().returns(aMethod)` otherwise but probably overkill here)

Comment (awebberCentrica) [reply tests/unit/user/route-test.js]: Even changing it to sinon.spy I get the same issue.

Comment (Leooo) [reply tests/unit/user/route-test.js]: ok my bad

**Review by Leooo — APPROVED** [review]

**Pull request merged**